format/view-spec.md

```diff
@@ -158,6 +173,59 @@ Each entry in `version-log` is a struct with the following fields:
 | _required_  | `timestamp-ms` | Timestamp when the view's `current-version-id` was updated (ms from epoch) |
 | _required_  | `version-id`   | ID that `current-version-id` was set to |
 
+#### Full identifier
+
+The full identifier holds a fully resolved reference for a table or view in the catalog.
+
+| Requirement | Field name     | Description |
+|-------------|----------------|-------------|
+| _required_  | `catalog` | A string specifying the catalog of the source table |
+| _required_  | `namespace`   | A list of namespace levels |
+| _required_  | `table`   | A string specifying the name of the source table |
+| _optional_  | `ref`   | Branch name of the source table that is being referenced in the view query  |
+
+When 'ref' is `null` or not set, it defaults to “main”. This field is to be ignored if the referenced entity is a view.
+
+### Materialized View Metadata stored as part of the Table Metadata
```

Comment:
Should we include **refresh-start-timestamp-ms** which is when the refresh is triggered and an approximation of the refresh state?

Reply:
That would work for me.

Reply:
I'm assuming the motivation here is to have a better approximation compared to the `timestamp-ms` field of the snapshot, which is taken at the end of the refresh operation. Is this correct?

Reply:
Yes, that's the motivation.  It's a quick way to say that the materialization is at most current time minus refresh-start-timestamp-ms stale.   It would also be useful if the view references non-Iceberg tables